StoreApp.BLL/Services/FilterProductsService.cs

```diff
@@ -0,0 +1,70 @@
+﻿using AutoMapper;
+using StoreApp.BLL.DTO;
+using StoreApp.DAL.Entities;
+using StoreApp.DAL.Intefaces;
+using StoreApp.DAL.Repositories;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace StoreApp.BLL.Services
+{
+    public class FilterProductsService
+    {
+        private IUnitOfWork DataBase { get; set; }
+
+        private IMapper config;
```

Comment:
We skipped this point but I guess it's better to keep this property and field together. Also we should follow one common style - declare properties or fields

Reply:
fixed